File: clients/dotnet/models/ContactModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace Avalara.AvaTax.RestClient
{
    /// <summary>
    /// A contact person for a company.
    /// </summary>
    public class ContactModel
    {
        /// <summary>
        /// The unique ID number of this contact.
        /// </summary>
        public Int32 id { get; set; }

        /// <summary>
        /// The unique ID number of the company to which this contact belongs.
        /// </summary>
        public Int32 companyId { get; set; }

        /// <summary>
        /// A unique code for this contact.
        /// </summary>
        public String contactCode { get; set; }

        /// <summary>
        /// The first or given name of this contact.
        /// </summary>
        public String firstName { get; set; }

        /// <summary>
        /// The middle name of this contact.
        /// </summary>
        public String middleName { get; set; }

        /// <summary>
        /// The last or family name of this contact.
        /// </summary>
        public String lastName { get; set; }

        /// <summary>
        /// Professional title of this contact.
        /// </summary>
        public String title { get; set; }

        /// <summary>
        /// The first line of the postal mailing address of this contact.
        /// </summary>
        public String line1 { get; set; }

        /// <summary>
        /// The second line of the postal mailing address of this contact.
        /// </summary>
        public String line2 { get; set; }

        /// <summary>
        /// The third line of the postal mailing address of this contact.
        /// </summary>
        public String line3 { get; set; }

        /// <summary>
        /// The city of the postal mailing address of this contact.
        /// </summary>
        public String city { get; set; }

        /// <summary>
        /// The state, region, or province of the postal mailing address of this contact.
        /// </summary>
        public String region { get; set; }

        /// <summary>
        /// The postal code or zip code of the postal mailing address of this contact.
        /// </summary>
        public String postalCode { get; set; }

        /// <summary>
        /// The ISO 3166 two-character country code of the postal mailing address of this contact.
        /// </summary>
        public String country { get; set; }

        /// <summary>
        /// The email address of this contact.
        /// </summary>
        public String email { get; set; }

        /// <summary>
        /// The main phone number for this contact.
        /// </summary>
        public String phone { get; set; }

        /// <summary>
        /// The mobile phone number for this contact.
        /// </summary>
        public String mobile { get; set; }

        /// <summary>
        /// The facsimile phone number for this contact.
        /// </summary>
        public String fax { get; set; }

        /// <summary>
        /// The date when this record was created.
        /// </summary>
        public DateTime? createdDate { get; set; }

        /// <summary>
        /// The User ID of the user who created this record.
        /// </summary>
        public Int32? createdUserId { get; set; }

        /// <summary>
        /// The date/time when this record was last modified.
        /// </summary>
        public DateTime? modifiedDate { get; set; }

        /// <summary>
        /// The user ID of the user who last modified this record.
        /// </summary>
        public Int32? modifiedUserId { get; set; }



        /// <summary>
        /// Convert this object to a JSON string of itself
        /// </summary>
        /// <returns>A JSON string of this object</returns>
        public override string ToString()
		{
            return JsonConvert.SerializeObject(this, new JsonSerializerSettings() { Formatting = Formatting.Indented });
		}
    }
}

File: clients/php/models/ContactModel.php
<?php 
namespace Avalara;
/*
 * AvaTax API Client Library
 *
 * (c) 2004-2016 Avalara, Inc.
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 *
 * @category   AvaTax client libraries
 * @package    Avalara.AvaTaxClient
 * @author     Ted Spence <ted.spence@avalara.com>
 * @author     Bob Maidens <bob.maidens@avalara.com>
 * @copyright  2004-2016 Avalara, Inc.
 * @license    https://www.apache.org/licenses/LICENSE-2.0
 * @version    2.16.12-30
 * @link       https://github.com/avadev/AvaTaxClientLibrary
 */


/**
 * A contact person for a company.
 */
final class ContactModel
{
    /**
     * @var Int32 The unique ID number of this contact.
     */
    public $id;

    /**
     * @var Int32 The unique ID number of the company to which this contact belongs.
     */
    public $companyId;

    /**
     * @var String A unique code for this contact.
     */
    public $contactCode;

    /**
     * @var String The first or given name of this contact.
     */
    public $firstName;

    /**
     * @var String The middle name of this contact.
     */
    public $middleName;

    /**
     * @var String The last or family name of this contact.
     */
    public $lastName;

    /**
     * @var String Professional title of this contact.
     */
    public $title;

    /**
     * @var String The first line of the postal mailing address of this contact.
     */
    public $line1;

    /**
     * @var String The second line of the postal mailing address of this contact.
     */
    public $line2;

    /**
     * @var String The third line of the postal mailing address of this contact.
     */
    public $line3;

    /**
     * @var String The city of the postal mailing address of this contact.
     */
    public $city;

    /**
     * @var String The state, region, or province of the postal mailing address of this contact.
     */
    public $region;

    /**
     * @var String The postal code or zip code of the postal mailing address of this contact.
     */
    public $postalCode;

    /**
     * @var String The ISO 3166 two-character country code of the postal mailing address of this contact.
     */
    public $country;

    /**
     * @var String The email address of this contact.
     */
    public $email;

    /**
     * @var String The main phone number for this contact.
     */
    public $phone;

    /**
     * @var String The mobile phone number for this contact.
     */
    public $mobile;

    /**
     * @var String The facsimile phone number for this contact.
     */
    public $fax;

    /**
     * @var DateTime? The date when this record was created.
     */
    public $createdDate;

    /**
     * @var Int32? The User ID of the user who created this record.
     */
    public $createdUserId;

    /**
     * @var DateTime? The date/time when this record was last modified.
     */
    public $modifiedDate;

    /**
     * @var Int32? The user ID of the user who last modified this record.
     */
    public $modifiedUserId;

}
